src/main/kotlin/racingcar/manager/RacingManager.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.manager
+
+import racingcar.car.Car
+
+class RacingManager(
+    private var carNum: Int,
+    private var attempts: Int
+) {
+
+    private var cars: List<Car> = listOf()
+
+    init {
+        init()
+    }
+
+    private fun init() {
+        cars = (0 until carNum).map { Car() }
+    }
```

Comment:
핵심 필드들이 메인 생성자가 되면 좋을거같아요.
```suggestion
class RacingManager constructor(private var cars: List<Car>, private val attempts: Int) {
    constructor(carNum: Int, attempts: Int):this((0 until carNum).map { Car() } , attempts)
```

아니면 팩토리를 만들어주는것도 방법인듯해요. 고민해보세요!